app/helpers/sessions_helper.rb

```diff
@@ -3,6 +3,10 @@ def log_in(user)
     session[:user_id] = user.id
   end
 
+  def logged_in?
+    !current_user.nil?
```

Comment:
Double negative .... makes it confusing to read. 

In ruby, all values except for nil and false are considered 'truthy' values. You can just return current_user from this method.

Another 'fancy' way of doing this would be to alias the `current_user` method to `:logged_in?`